core/templates/pages/blog-dashboard-page/blog-dashboard-tile/blog-dashboard-tile.component.ts

```diff
@@ -49,7 +49,7 @@ export class BlogDashboardTileComponent implements OnInit {
 
   getDateStringInWords(naiveDate: string): string {
     return dayjs(
-      naiveDate.split(',')[0], 'MM-DD-YYYY').format('MMMM D, YYYY');
+      naiveDate.split(',')[0], 'MM-DD-YYYY').format('MMM D, YYYY');
```

Comment:
Why this change?

Reply:
It instead of displaying the full month, displays it in short form like  instead of 'September' it will be 'Sept' now. It was looking clumsy and was occupying much more space. Also in the mocks:
The first row has only 'Feb' ,  So changed it accordingly
![Screenshot from 2021-09-07 19-18-41](https://user-images.githubusercontent.com/68547101/132356388-bc025770-2304-4773-a38a-24e6652568f8.png)